tool-timber/src/main/kotlin/com/infinum/sentinel/ui/logger/LoggerAdapter.kt

```diff
@@ -5,12 +5,16 @@ import android.view.ViewGroup
 import androidx.recyclerview.widget.ListAdapter
 import com.infinum.sentinel.SentinelFileTree
 import com.infinum.sentinel.databinding.SentinelItemLogBinding
+import java.text.SimpleDateFormat
+import java.util.Locale
 
 internal class LoggerAdapter(
     private val onListChanged: (Boolean) -> Unit,
     private val onClick: (SentinelFileTree.Entry) -> Unit
 ) : ListAdapter<SentinelFileTree.Entry, LoggerViewHolder>(LoggerDiffUtil()) {
 
+    private val dateTimeFormat = SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss. SSSZ", Locale.getDefault())
```

Comment:
Here too, and it also makes me wonder if we could just share this pattern or formatted between the two files 😄 

```suggestion
    private val dateTimeFormat = SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSZ", Locale.getDefault())
```

Reply:
Yea, of course